proposals/p1964.md

```diff
@@ -90,8 +91,9 @@ We will not support:
 -   Multi-line literals
 -   "raw" literals (using #'x'#)
 -   Empty character literals (`''`)
--   ASCII Control codes (0...31), except for `\` when used for an escape
-    sequence. See
+-   ASCII Control codes (0...31), except when specified with an escape sequence.
+-   Whitespace characters other than word space: tab, line feed, carriage
+    return, form feed, and vertical tab. See
     [String Literals: Escaping Sequence](https://github.com/carbon-language/carbon-lang/blob/trunk/proposals/p0199.md#escape-sequences-1)
 
 ## Details
```

Comment:
Just a note to add the conclusion from https://discord.com/channels/655572317891461132/1028052594070929499/1028081432251338833
to this section.